src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ScoreSystem.java

```diff
@@ -38,50 +23,18 @@
 import org.terasology.ligthandshadow.componentsystem.events.ScoreUpdateFromServerEvent;
 import org.terasology.lightandshadowresources.components.LASTeamComponent;
 import org.terasology.lightandshadowresources.components.WinConditionCheckOnActivateComponent;
-import org.terasology.nui.ControlWidget;
-import org.terasology.nui.databinding.ReadOnlyBinding;
-import org.terasology.nui.widgets.UILabel;
 
 @RegisterSystem(RegisterMode.AUTHORITY)
-@Share(ScoreSystem.class)
 public class ScoreSystem extends BaseComponentSystem {
 
     @In
     private InventoryManager inventoryManager;
     @In
-    private NUIManager nuiManager;
-    @In
     private EntityManager entityManager;
 
     private int redScore;
     private int blackScore;
 
-    @Override
-    public void postBegin() {
-        // Sets score screen bindings
-        ControlWidget scoreScreen = nuiManager.getHUD().getHUDElement("LightAndShadow:ScoreHud");
-        UILabel blackScoreArea = scoreScreen.find("blackScoreArea", UILabel.class);
-        blackScoreArea.bindText(new ReadOnlyBinding<String>() {
-            @Override
-            public String get() {
-                return String.valueOf(blackScore);
-            }
-        });
-        UILabel redScoreArea = scoreScreen.find("redScoreArea", UILabel.class);
-        redScoreArea.bindText(new ReadOnlyBinding<String>() {
-            @Override
-            public String get() {
-                return String.valueOf(redScore);
-            }
-        });
-    }
-
-    @Override
-    public void initialise() {
-        // Displays score UI on game start
-        nuiManager.getHUD().addHUDElement("ScoreHud");
-    }
```

Comment:
Did you miss adding the file you moved this to to the commit? 🤔

Reply:
This is already a part of ClientScoreSystem.

Reply:
Although those components are a part of those prefabs, no system is utilizing them so they are not really required.

Reply:
> Although those components are a part of those prefabs, no system is utilizing them so they are not really required. We should remove these components from those prefabs too.

Have you compared carrying a flag in-game with and without the PR?

Reply:
Yup I tested everything and it worked fine, while picking up the flags the particles were rendered. Will test it in multiplayer along with the LAS staff just in case.